packages/commonwealth/server/migrations/20230616214402-remove-duplicate-eth-chain-node.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.bulkUpdate(
+        'Chains',
+        {
+          chain_node_id: 37,
+        },
+        {
+          chain_node_id: 1285,
+        }
+      );
+
+      await queryInterface.bulkDelete('ChainNodes', {
+        id: 1285,
```

Comment:
What ChainNode does this id refer to?

Reply:
1285 is the duplicate Ethereum chain node that was added after someone created an ERC20 community with the incorrect hard-coded chain node URL (wss alchemy URL).